src/ui/main_menu.rs

```diff
@@ -74,95 +64,87 @@ pub fn setup_main_menu_system(
+                                        aspect_ratio: Some(1920.0 / 1080.0),
+                                        justify_content: JustifyContent::Center,
+                                        align_items: AlignItems::Center,
+                                        ..default()
                                     },
-                                )
-                                    .with_justify(JustifyText::Center),
-
-                                ..default()
-                            });
+                                    image: ui_assets.thetawave_logo_image.clone().into(),
+                                    texture_atlas: ui_assets.thetawave_logo_layout.clone().into(),
+                                    ..default()
+                                })
+                                .insert(AnimationComponent {
+                                    timer: Timer::from_seconds(0.1, TimerMode::Repeating),
+                                    direction: AnimationDirection::Forward,
+                                });
                         });
-
                     parent
-                        .spawn(ImageBundle {
-                            image: asset_server
-                                .load(if **playing_on_arcade {
-                                    "texture/start_game_prompt_arcade.png"
-                                } else {
-                                    "texture/start_game_prompt_keyboard.png"
-                                })
-                                .into(),
+                        .spawn(NodeBundle {
                             style: Style {
-                                width: Val::Px(400.0),
-                                height: Val::Px(100.0),
-                                margin: UiRect {
-                                    bottom: Val::Percent(10.0),
-                                    top: Val::Auto,
-                                    right: Val::Auto,
-                                    left: Val::Auto,
-                                },
+                                width: Val::Percent(100.0),
+                                height: Val::Percent(50.0),
+                                flex_direction: FlexDirection::Column,
                                 justify_content: JustifyContent::Center,
-                                ..Default::default()
+                                align_items: AlignItems::Center,
+                                ..default()
                             },
-                            ..Default::default()
+                            ..default()
                         })
-                        .insert(BouncingPromptComponent {
-                            flash_timer: Timer::from_seconds(2.0, TimerMode::Repeating),
-                            is_active: true,
+                        .with_children(|parent| {
+                            parent.spawn_menu_button(
```

Comment:
Im gonna try to get arrow keys/gamepad input to work here. Otherwise we can't start the game on the arcade.